frank/radial_fitters.py

```diff
@@ -716,6 +717,8 @@ def fit(self, u, v, V, weights=1):
         pi_old = 0
         while (np.any(np.abs(pi - pi_old) > self._tol * pi) and
                count <= self._max_iter):
+            #print('\r    Fit iteration {}'.format(count), end='', flush=True)
```

Comment:
```suggestion
            if verbose:
                print('\r    FrankFitter iteration {}'.format(count), end='', flush=True)
```

If you want to add it in, I think this is fine. You'd need to add verbose as a kwarg to fit

Reply:
cool, good point, I've updated in `radial_fitters.py` and the call in `fit.py`